src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -1149,7 +1149,6 @@ public void HandleDatabaseValueChanged(string type, object changeState)
 
                 periodicBackup.BackupStatus ??= new PeriodicBackupStatus();
                 periodicBackup.BackupStatus.DelayUntil = state.DelayUntil;
-                ScheduleNextBackup(periodicBackup, state.DelayUntil.Subtract(DateTime.UtcNow), lockTaken: false);
```

Comment:
How does it fix the issue?

Reply:
The problem occurs because the `ScheduleNextBackup` method is executed twice when the `delay` is performed:

1. https://github.com/ravendb/ravendb/blob/900eb7b6b4c6294e9bd251566fea3028d5386663/src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs#L1152
2. https://github.com/ravendb/ravendb/blob/900eb7b6b4c6294e9bd251566fea3028d5386663/src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs#L603

Inside the `ScheduleNextBackup` method, we manage the `ConcurrentBackupsCounter` by indicating the completion of the backup process, which decreases the number of concurrent backups. However, due to the **erroneous double execution** of the `ScheduleNextBackup` method during the delay of a backup task, the `ConcurrentBackupsCounter` incorrectly decreases its value by 2 instead of 1.